### ISSUE TITLE
Install ring.com floodlight on brick wall stuck as 1) drill hole not going well 2) hole made with 3/16 in drill bit too tight

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#600]
As per title.

**Comment by 130s** [comment]
# What worked ok

With Ryobi handheld drill, and 3/16 inch's [Ryobi's masonry drill bit (3-piece) (homedepot.com)](https://www.homedepot.com/p/RYOBI-Impact-Rated-Masonry-Set-3-Piece-AR2043/308112745#product-overview), I was able to make 1 drill hole after several minutes of drilling.

# Problems detailed

## Issue-1. drill hole not going well
Unlike the one hole, other drill holes don't go well. It'd take hours I'm afraid if I had continued. Also I'm afraid the motor would burn down if I continue drilling that long, with extensive force I exert by hands.

## Issue-2. Hole may be too small
The masonry anchor barely fits into the only drill hole I managed to make. Afraid if I'd continue hammer it down the anchor may collapse.

Anchor says 6x30. Does this mean??:
- 6 mm diameter
- 30 mm length

![PXL_20211105_202432900](https://user-images.githubusercontent.com/1840401/140575956-d95ad9fd-3b13-4b46-a62f-07a5839ada38.jpg)

If above is correct then I was wrong in using 3/16 in. drill bit, as 6mm is about 0.24 inch, which 1/4 inch is closer than 3/16.

**Comment by 130s** [comment]
Conclusion: Try again with 1/4 inch drill bit.

**Comment by 130s** [comment]
Asked [diy.stackexchange.com#q238293](https://diy.stackexchange.com/questions/238293/not-sure-what-drill-bit-to-use-for-6mm-masonry-anchor-into-brick).

**Comment by 130s** [comment]
Done. As I've updated [diy.stackexchange.com#q238293](https://diy.stackexchange.com/questions/238293/not-sure-what-drill-bit-to-use-for-6mm-masonry-anchor-into-brick), I ended up using the drill bit of the right size that came with the outside camera.

**Drilling 2 holes on the brick took long, more than 30 minutes I think, using the automated drill. This is something to avoid in the future if possible**.

I unscrewed something remained from previous home owner. Indeed there's masonry anchor used.
<img src="https://user-images.githubusercontent.com/1840401/153778552-fd5eb6af-2695-424e-b421-69e5f9b35ebd.jpg" width=55% />

Base screwed in, but this turned out wrong. I noticed I did backside front. So I re-did (took no pic of the right config).
<img src="https://user-images.githubusercontent.com/1840401/153778559-b6b51f83-584d-4a18-b104-60bc2bcb5e41.jpg" width=55% />
